src/static/index.html

```diff
@@ -15,35 +15,36 @@
     <script type="text/javascript" src="https://igv.org/web/release/1.0.6/igv-1.0.6.min.js"></script>
     <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
     <script type="text/javascript">
-      google.charts.load('current', {'packages':['timeline']});
+        google.charts.load('current', {'packages': ['timeline']});
     </script>
     <!-- end IGV -->
     <style>
-      body {
-        margin: 0;
-        font-family: 'Roboto', sans-serif;
-        -webkit-font-smoothing: antialiased;
-        -moz-font-smoothing: antialiased;
-        -o-font-smoothing: antialiased;
-        background-color: #f4f4f4;
-      }
-      *:disabled {
-        cursor: not-allowed;
-      }
+        body {
```

Comment:
Same here. Two spaces seems enough to distinguish while preserving valuable horizontal space.